### DISCUSSION TITLE
사용자 명을 바꿀 수 있을 것처럼 나오는데 변경이 안됩니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#27] [제보하기]
사용자 목록에서 나를 선택하면 사용자 명을 바꿀 수 있을 것처럼 나오는데 변경이 안됩니다.

<img width="508" alt="image" src="https://github.com/metamong-friends/metamong-friends.github.io/assets/24870098/be8de4bf-1350-4f2e-82ce-7c4cb5e8bacd">

**Comment by ACT-TJ** [comment]
네. 변경되는 것이 맞는데 변경되지 않았다면 Bug 로 보여지네요.
현재 테스트 시 수정은 잘 되는데 포커스가 한번씩 안되는 상황은 보여집니다.

혹시 해당 답변 확인하신다면 다시 한 번 수정 시도해 보시고 답변 주시면 감사하겠습니다^^
저희도 기능 확인 및 재현해 보고 다시 피드백 드리도록 하겠습니다.

**Reply by DolDolDol** [reply]
지금 잘 되는 것 확인했습니다!

**Reply by shlee303** [reply]
다행이네요~! 감사합니다~
